noir-projects/noir-contracts/contracts/claim_contract/src/main.nr

```diff
@@ -4,11 +4,12 @@ use dep::aztec::macros::aztec;
 pub contract Claim {
```

Comment:
Changes in this contract are mainly related to [this comment](https://github.com/AztecProtocol/aztec-packages/pull/11904#discussion_r1954122823).

Reply:
This kind of thing definitely should've happened elsewhere. The changeset here is already very large as-is.